base/src/main/java/io/spine/base/Environment.java

```diff
@@ -159,6 +192,27 @@ public Environment register(EnvironmentType environmentType) {
         return this;
     }
 
+    /**
+     * Remembers the specified environment type, allowing {@linkplain #is(Class) to
+     * determine whether it's enabled} later.
+     *
+     * <p>The specified {@code type} must have a parameterless constructor.
+     * {@linkplain EnvironmentType#enabled() activity} of the specified environment type is going
```

Comment:
`activity` should start from the capital letter. But I am not sure I understand this sentence:
```
activity of the specified environment type is going to be checked against an instance created by invoking the parameterless constructor.
```
Please rewrite into two or more _simple_ sentences.